packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/add/logs-encryption-keys-add.controller.js

```diff
@@ -0,0 +1,78 @@
+export default class LogsEncryptionKeysAddCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $stateParams,
```

Comment:
Same as https://github.com/ovh/manager/pull/8249#discussion_r998357121

Reply:
Question about this one: what should I move to the routing's `resolve`:
- the `$stateParams` itself ?
- or the values in `$stateParams` that use in my controller ?

For instance here in my controller I only use it for `this.serviceName = this.$stateParams.serviceName;`.
If I get it right, I should resolve the `serviceName` directly in the routing's `resolve` ?

Reply:
Fixed, now resolving `serviceName` in routing's `resolve`

Reply:
Yes, I see you have implemented the correct way using `$transition$.params()`. Good find :)